package.json

```diff
@@ -25,6 +25,7 @@
   "dependencies": {
     "babel-runtime": "^6.26.0",
     "base64-js": "^1.2.1",
+    "moment": "^2.20.1",
```

Comment:
Hey, have you heard of/considered [`date-fns`](https://date-fns.org/)? I've found it easier to use and won't bloat your bundle 😃

Reply:
I haven't, but will now.  Thanks!